book/src/users/startup.md

```diff
@@ -22,7 +22,7 @@ cargo run -p trin -- --help
 ```
 ### Bootnodes
 
-To quickly connect to the testnet use the `--bootnodes default` flag.
+Trin automatically connects to the testnet bootnodes. Use the `--bootnodes` cli flag to connect to a specific node or to none.
```

Comment:
I think we should start moving away from "testnet" terminology...
```suggestion
Trin automatically connects to a set of hardcoded bootnodes. Use the `--bootnodes` cli flag to connect to a specific node or to none.
```

Reply:
as in moving away from "testnet" towards "mainnet"? The assumption in your suggested change being that the hardcoded bootnodes are on the mainnet? or maybe rather the "Portal Network" to avoid confusion with the execution network

Reply:
- "portal network"
- "live network"
- "main network"

Basically, we just aren't operating in a "testnet".  It's the main/live/official/real network.